config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -93,7 +93,7 @@ IF(PYTHONINTERP_FOUND)
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
                      NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
+                           boost_python${PYTHON_VERSION_MAJOR}                             # Mac with Python 3
```

Comment:
Yes, this happened in the CI. We can keep `mt`, but I suggest we move it down the list. We should prefer the non-`mt` version as we have done in other cases here first for consistency.